packages/datadog-plugin-jest/test/circus.spec.js

```diff
@@ -40,7 +40,8 @@ describe('Plugin', function () {
         .reply(200, 'OK')
 
       tracer = require('../../dd-trace')
-      return agent.load(['jest', 'fs', 'http']).then(() => {
+      return agent.load(['jest', 'fs', 'http'], { service: 'jest-test-circus' }).then(() => {
+        tracer._tracer._service = 'jest-test-circus'
```

Comment:
this is only so that https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/format.js#L96 is run and the language tag is added (so as not to change every test)

Reply:
Since the option is set, why is this needed unless there is a bug?

Reply:
I don't know why the language tag is only added in the case in which the service is the same, so I'm not sure.

Reply:
What I mean is that by setting the option on the line above, the service name should already be `jest-test-circus` so there should be no need to manually override the private property. If that's not the case there might be a bug in the plugin.

Reply:
do other plugins set `tracer._service` if passed a `service` in their `config` ? 
Maybe the problem is checking `service` in `traces[0][0].meta`  instead of `'service.name'`

Reply:
If the plugin config is working properly, then the `service.name` tag should match the tracer service without having to explicitly change the tracer service in the test.